src/arviz_plots/visuals/__init__.py

```diff
@@ -77,8 +84,12 @@ def ecdf_line(values, target, backend, **kwargs):
     return plot_backend.line(values.sel(plot_axis="x"), values.sel(plot_axis="y"), target, **kwargs)
 
 
-def fill_between_y(da, target, backend, *, x=None, y_bottom=None, y_top=None, **kwargs):
+def fill_between_y(da, target, backend, *, x=None, y_bottom=None, y=None, y_top=None, **kwargs):
     """Fill the region between to given y values."""
+    if y_bottom is None and y:
```

Comment:
Side comment is that `if y_bottom is None and y` would evaluate `y` as a boolean, thus, `y_bottom=None, y=0` would not enter the if. Not relevant with the other behavioural comment though